lib/components/position_component.dart

```diff
@@ -25,6 +24,9 @@ abstract class PositionComponent extends BaseComponent {
   /// The position of this component on the screen (relative to the anchor).
   Vector2 position = Vector2.zero();
 
+  /// If the component has a [PositionComponent] as a parent it will be set here
+  PositionComponent _positionParent;
```

Comment:
should we just store BaseComponent parent and we can check `is Position` where it's used? having the parent can be useful for other things

Reply:
That is a good idea! Maybe later we can expose a getter for the parent too.

Reply:
Fixed!